src/lib/util/Models/ApiTransform.ts

```diff
@@ -42,9 +42,9 @@ export interface FlattenedGuild {
 	splash: string | null;
 	region: string;
 	features: GuildFeatures[];
-	applicationID: string;
+	applicationID: string | null;
 	afkTimeout: number;
-	afkChannelID: string;
+	afkChannelID: string | null;
 	systemChannelID: string | null;
 	embedEnabled: boolean;
```

Comment:
Wouldn't it be better for this interface to extend DJS' Guild class, or if necessary, use Pick or Omit to grab those fields it is currently duplicating?

Reply:
No, `FlattenGuild`'s output is strictly fixed, it doesn't care about the properties discord.js's `Guild` has.